src/sidebar/build-thread.js

```diff
@@ -157,7 +176,7 @@ function mapThread(thread, mapFn) {
  * Return a sorted copy of an array of threads.
  *
  * @param {Array<Thread>} threads - The list of threads to sort
- * @param {(Annotation,Annotation) => boolean} compareFn
+ * @param {(a: Annotation, b: Annotation) => boolean} compareFn
```

Comment:
Function parameters in types need to have names, although it doesn't matter what.